src/tests/ftest/soak/utils.py

```diff
@@ -198,22 +196,16 @@ def get_remote_dir(self, source_dir, dest_dir, host_list, shared_dir=None,
         # copy the local logs and the logs in the shared dir to avocado dir
         for directory in [source_dir, shared_dir]:
             command = "/usr/bin/cp -R -p {0}/ \'{1}\'".format(directory, dest_dir)
-            try:
-                run_command(command, timeout=30)
-            except DaosTestError as error:
-                raise SoakTestError("<<FAILED: job logs failed to copy>>") from error
+            run_command(command, timeout=30, raise_exception=False)
+
     if rm_remote:
         # remove the remote soak logs for this pass
         command = "/usr/bin/rm -rf {0}".format(source_dir)
         slurm_utils.srun(NodeSet.fromlist(host_list), command, self.srun_params)
         # remove the local log for this pass
         for directory in [source_dir, shared_dir]:
             command = "/usr/bin/rm -rf {0}".format(directory)
-            try:
-                run_command(command)
-            except DaosTestError as error:
-                raise SoakTestError(
-                    "<<FAILED: job logs failed to delete>>") from error
+            run_command(command, raise_exception=False)
```

Comment:
We should reinstate the SoakTestError raising for non-tearDown calling and instead use a try .. except .. in the pre_tear_down() method to capture any errors and perform the appropriate action (log a message, or add the error to the list of errors to be reported at the end of tearDown, etc.)